mscore/shortcut.cpp

```diff
@@ -182,6 +182,15 @@ Shortcut Shortcut::_sc[] = {
          Icons::redo_ICON,
          Qt::ApplicationShortcut
          },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL,
+         "view-options",
+         QT_TRANSLATE_NOOP("action","View Options"),
+         0,
```

Comment:
Please change that `0,` into `QT_TRANSLATE_NOOP("action","View options"),`, needed for zthe shortcuts dialog (and as a tooltip) and there in sentence case

Reply:
This comment got me thinking. The zoom box and view mode box don't really belong in the shortcuts list at all. There are already shortcuts for various zoom and page mode actions. So I've slightly changed tack. (See below.)